### DISCUSSION TITLE
Nuxt 3 - Private beta

### DISCUSSION THREAD

**Discussion by atinux** [nuxt/nuxt#18362] [General]
Hello everyone 👋 

If you have access to this repository, this mean you are part of the private beta program in order to help us make Nuxt 3 stable before opening to the whole community 😊 

The private beta goal is for **new projects only** so we can focus on making Vue 3, SSR, Webpack 5 and Vite support the best it can in term of developer experience.

The documentation is live on https://v3.nuxtjs.org.
Username is `nuxt` and password is `ilovenuxt`.

We highly recommend to check:
- [Open issues](https://github.com/nuxt/framework/issues) to understand the current work.
- [Readme](https://github.com/nuxt/framework/blob/main/README.md) for instructions

Thank you very much ❤️

**Comment by leo** [comment]
Amazing! Trying it out at the moment!

Are you planning to replace the demo that the documentation currently talks about with [this](https://github.com/nuxt/starter/tree/v3) (@danielroe shared this)? And if so, when? It seems like the current demo is only a blank white screen with a bit of text.

**Reply by atinux** [reply]
Hey @leo 

Yes we do plan to update the demo once we hit a stable version 😊

**Reply by pi0** [reply]
You can track it here: https://github.com/nuxt/framework/issues/596 -- There is a welcome page but hidden when there is a starter `app.vue` file
